src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form"
+      action="#"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+        class="search-input search-input--big"
```

Comment:
The attributes in the <input> tag should also follow the same formatting rule as the <form> tag, with each attribute on a new line and properly indented.